src/components/subHeader/components/descriptionAddress/DescriptionAddress.js

```diff
@@ -0,0 +1,41 @@
+// Libraries
+import PropTypes from 'prop-types'
+
+// reader
+import addressReader from "../../readers/addressReader";
+
+
+function DescriptionAdress(props){
+
+    const { address } = props;
+
+    const name = addressReader.name(address);
+    const type = addressReader.type(address);
+    const city = addressReader.city(address);
+    
+    return (
+        <>
+            <p className="description-heading">{name}</p>
+            <p className="description-address">{type}</p>
```

Comment:
can you try to change the typography of type and city, use lightText, that would make it more elegant